src/systems/filecoin_mining/sector/posting.id

```diff
@@ -55,15 +55,18 @@ type OnChainPoStVerifyInfo interface {
 }
 
 type OnChainElectionPoStVerifyInfo struct {
-    Candidates  [PoStCandidate]
-    Proofs      [PoStProof]
-    Randomness  PoStRandomness
+    // There should be one RegisteredProof for each PoSt Candi
+    RegisteredProofs  [RegisteredProof]
```

Comment:
For follow-up, talk to @zixuanzh about whether we should store the RegisteredProof for each sector on-chain, meaning we could omit it here.